### ISSUE TITLE
How to remove the num_virtual_token virtual prompt predictions from the left side of the model outputs during evaluations.

### ISSUE THREAD

**Issue by zzh-SJTU** [huggingface/peft#482]
I am using Prompt-Tuning for SEQ_2_SEQ task. I found that the model outputs.logits size is num_virtual_token + max_length. 

with torch.no_grad():
            outputs = model(**batch)
torch.argmax(outputs.logits, -1).detach().cpu().size() 

How to remove the num_virtual_token virtual prompt predictions from the left side of the model outputs during evaluations.

**Comment by zzh-SJTU** [comment]
Should we remove them after decoding or before decoding?
